ocpp/messages.py

```diff
@@ -101,6 +104,10 @@ def get_schema(message_type_id, action, ocpp_version, parse_float=float):
     is used to parse floats. It must be a callable taking 1 argument. By
     default it is `float()`, but certain schema's require `decimal.Decimal()`.
     """
+    warnings.warn(
+        "Depricated as of 0.8.1. Please use `ocpp.messages.get_validator()`."
```

Comment:
typo, it's "Deprecated"

Reply:
He has deprecated your comment @Roger :P

Reply:
@OrangeTux fix that tiny typo so that I can approve this peral

Reply:
"peral"

Reply:
hahaha

Reply:
that was the typo, I was talking about ;)